pkg/util/machines.go

```diff
@@ -135,24 +135,31 @@ func IsControlPlaneMachine(machine metav1.Object) bool {
 	return ok
 }
 
+// max calculates the maximum integer for two integer values.
+func max(valueOne, valueTwo int) int {
```

Comment:
we should use `k8s.io/utils/integer` instead of this

Reply:
Didn't know this existed. Thanks!